pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -409,7 +409,7 @@ func (ctrl *PersistentVolumeController) syncUnboundClaim(claim *v1.PersistentVol
 			klog.V(4).Infof("synchronizing unbound PersistentVolumeClaim[%s]: volume %q requested and found: %s", claimToClaimKey(claim), claim.Spec.VolumeName, getVolumeStatusForLogging(volume))
 			if volume.Spec.ClaimRef == nil {
 				// User asked for a PV that is not claimed
-				// OBSERVATION: pvc is "Pending", pv is "Available"
+				// OBSERVATION: pvc is "Pending"
```

Comment:
PV can be in any state. Typically, PV is in released state after the PVC is deleted.